ete4/smartview/renderer/faces.py

```diff
@@ -1542,7 +1551,7 @@ def draw(self, drawer):
 class PieChartFace(CircleFace):
 
     def __init__(self, radius, data, name="",
-            padding_x=0, padding_y=0):
+            padding_x=0, padding_y=0, tooltip=None):
 
         Face.__init__(self, name=name,
                 padding_x=padding_x, padding_y=padding_y)
```

Comment:
I think we need to pass the `tooltip` argument to the constructor of the parent class (also, I think we should call `super()` instead of `Face`). This way it would actually use the tooltip when rendering. So it would be:

```py
super().__init__(self, name=name, padding_x=padding_x, padding_y=padding_y, tootltip=tooltip)
```